.travis.yml

```diff
@@ -1,6 +1,17 @@
+sudo: false
+os:
+  - linux
 language: node_js
 node_js:
-  - "0.10"
   - "0.12"
   - "4"
-  - "node"
+  - "6"
+env:
+  matrix:
+    - TEST_SUITE=unit
+matrix:
+  include:
+    - os: linux
+      node_js: "6"
+      env: TEST_SUITE=lint
+script: npm run $TEST_SUITE
```

Comment:
Seems complicated, `standard` runs in < 5 seconds and you can just look at the error output to figure out what is wrong. But I guess this is fine...